cygnus-twitter/README.md

```diff
@@ -1 +1,296 @@
-TBD
+#<a name="top"></a>Cygnus-twitter
+[![License badge](https://img.shields.io/badge/license-AGPL-blue.svg)](https://opensource.org/licenses/AGPL-3.0)
```

Comment:
Badges are finally added to the main README, so they are not necessary here. In any case, add the cygnus-twitter badge to that main README.